app/drivers/tools/repair/python/Brafar.py

```diff
@@ -60,19 +61,4 @@ def analyse_output(self, dir_info, bug_id, fail_list):
         self.emit_normal("reading output")
 
         # extract information from output log
```

Comment:
To relate with the email - this is the place where you would update the patch count

Reply:
Thank you for your kind reply !!!!

Reply:
I may have misunderstood the process of output analysis before. If all the analysis is built by me, how do you validate my output patches?

Reply:
Hii, no problem :)